app/controllers/job_runs_controller.rb

```diff
@@ -21,7 +21,17 @@ def show
     @job_run = JobRun.find(params[:id])
   end
 
-  def download
+  def download_log
+    @job_run = JobRun.find(params[:id])
+    if @job_run.progress_log_file
+      send_file @job_run.progress_log_file
+    else
+      flash[:warning] = 'Job has not started. Please check back later.'
+      render 'show'
+    end
+  end
+
+  def download_report
```

Comment:
this renamed endpoint is now used to download the log file, add a new endpoint to download reports